src/test/kotlin/org/rust/ide/template/postfix/WhileNotPostfixTemplateTest.kt

```diff
@@ -0,0 +1,167 @@
+package org.rust.ide.template.postfix
+
+class WhileNotPostfixTemplateTest : PostfixTemplateTestCase(WhileNotExpressionPostfixTemplate()) {
+    fun testNumber() =
+        doTestNotApplicable(
+            """
+            fn main() {
+                let a = 4;
+                a.whilenot<caret>
+            }
+            """
+        )
+
+    fun testNumberCall() =
+        doTestNotApplicable(
+            """
+            fn func() -> i32 {
+                1234
+            }
+
+            fn main() {
+                func().whilenot<caret>
+            }
+            """
+        )
+
+    fun testSimple() =
+        doTest(
+            """
+            fn main() {
+                let a = 4 == 2;
+                a.whilenot<caret>
+            }
+            """
+            ,
+            """
+            fn main() {
+                let a = 4 == 2;
+                while !a {<caret>}
+            }
+            """
+        )
+
+    fun testSimpleNegatedExpr() =
```

Comment:
Hm, still can't see it. This is `testSimple`:

```
+            """
 +            fn main() {
 +                let a = 4 == 2;
 +                a.whilenot<caret>
 +            }
 +            """
 +            ,
 +            """
 +            fn main() {
 +                let a = 4 == 2;
 +                while !a {<caret>}
 +            }
 +            """
```

this is `testSImpleNegatedExpr`:

```
+            """
 +            fn main() {
 +                let a = 4 == 2;
 +                a.whilenot<caret>
 +            }
 +            """
 +            ,
 +            """
 +            fn main() {
 +                let a = 4 == 2;
 +                while !a {<caret>}
 +            }
 +            """
```